src/main/java/uk/gov/hmcts/reform/docassembly/DocAssemblyApi.java

```diff
@@ -3,19 +3,27 @@
 import org.springframework.cloud.openfeign.FeignClient;
 import org.springframework.http.HttpHeaders;
 import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestHeader;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import uk.gov.hmcts.reform.docassembly.domain.DocAssemblyRequest;
 import uk.gov.hmcts.reform.docassembly.domain.DocAssemblyResponse;
 import uk.gov.hmcts.reform.docassembly.healthcheck.InternalHealth;
 
+import static org.springframework.http.HttpHeaders.CONTENT_TYPE;
+import static org.springframework.http.MediaType.APPLICATION_JSON_UTF8_VALUE;
+
 @FeignClient(name = "doc-assembly-api", url = "${doc_assembly.url}")
 public interface DocAssemblyApi {
+    @GetMapping(
+        value = "/health",
+        headers = CONTENT_TYPE + "=" + APPLICATION_JSON_UTF8_VALUE
```

Comment:
do you really need a content type on this?
I think it could just be:
```suggestion
        @GetMapping("/health")
```

Reply:
I blame the power of `copypaste`